File: Classes/EVEOnlineAPI/EVE/EVECharacterAffiliation.m
//
//  EVECharacterAffiliation.m
//  EVEOnlineAPI
//
//  Created by Артем Шиманский on 11.08.15.
//
//

#import "EVECharacterAffiliation.h"

@implementation EVECharacterAffiliation

+ (NSDictionary*) scheme {
	static NSDictionary* scheme = nil;
	if (!scheme)
		scheme = @{@"characters":@{@"type":@(EVEXMLSchemePropertyTypeObject), @"class":[EVEAPIKeyInfoCharactersItem class]}};
	return scheme;
}

@end

File: EVEAPI/EVEAPI/EVECharacterAffiliation.swift
//
//  EVECharacterAffiliation.swift
//  EVEAPI
//
//  Created by Artem Shimanski on 30.11.16.
//  Copyright © 2016 Artem Shimanski. All rights reserved.
//

import UIKit

public class EVECharacterAffiliation: EVEResult {
	public var characters: [EVEAPIKeyInfoCharacter] = []
	
	public required init?(dictionary:[String:Any]) {
		super.init(dictionary: dictionary)
	}
	
	public required init?(coder aDecoder: NSCoder) {
		super.init(coder: aDecoder)
	}
	
	override public func scheme() -> [String:EVESchemeElementType] {
		return [
			"characters":EVESchemeElementType.Rowset(elementName: nil, type: EVEAPIKeyInfoCharacter.self, transformer: nil)
		]
	}
}
